Druid/DRBytecodeIRGenerator.class.st

```diff
@@ -303,6 +305,27 @@ DRBytecodeIRGenerator >> interpretPushWith: aRBMessageNode [
 	^ self addInstructionWithNoResultFrom: aRBMessageNode instructionKind: DRPush operands: { receiver }
 ]
 
+{ #category : #'special-cases' }
+DRBytecodeIRGenerator >> interpretRememberWith: aRBMessageNode [
+
+	| obj |
+	obj := (self visitOperand: aRBMessageNode arguments first)
+		       simpleConstantFold.
+
+	^ (self
+		   addInstructionWithNoResultFrom: aRBMessageNode
+		   instructionKind: DRCall
+		   operands: {
+				   'ceStoreCheckTrampoline'.
+				   obj }) beMapped saveLinkReg
+
+	"	
+	cogit 
+		evaluateTrampolineCallBlock: [cogit CallRT: ceStoreCheckTrampoline]
+		protectLinkRegIfNot: inFrame.
```

Comment:
Ok, we should review this later, I wonder why we need a special trampoline call.

Reply:
Now we are generating something like that, I forget the comment.